### PR TITLE
Update release.yaml

### PR DESCRIPTION
**Pull request by junkil-park**
### Description
Update `release.yaml` for v1.8

### CODE REVIEW

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -1,75 +1,64 @@
...
+      title: "AIP-55: Generalize Transaction Authentication and Support Arbitrary K-of-N MultiKey Accounts"
+      description: "AIP-55: Generalize Transaction Authentication and Support Arbitrary K-of-N MultiKey Accounts."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/267"
     execution_mode: MultiStep
     update_sequence:
       - FeatureFlag:
           enabled:
-            - bulletproofs_natives
-  - name: step_6_enable_storage_deletion_refund
+            - single_sender_authenticator
```
Comment (davidiw) [inline aptos-move/aptos-release-builder/data/release.yaml:54]: should we combine this was the secp? since it is the same feature flag.

Comment (junkil-park) [reply aptos-move/aptos-release-builder/data/release.yaml]: @runtian-zhou also confirmed that they should be combined if they use the same feature flag. 

Would this be a proper way to combine the information from two steps? https://github.com/aptos-labs/aptos-core/pull/10679/files#diff-f0942a7d2fc980b7d9d90ec193ec4e9214cdb34c3e3c1d59fa7d5507325b1574R15

I am not sure if `discussion_url: https://url1, https://url2` would end up generating a proper link to the discussions. (@movekevin , @perryjrandall ?)

**Review by davidiw — APPROVED** [review]

**Review by movekevin — COMMENTED** [review]
All look correct

File: aptos-move/aptos-release-builder/data/release.yaml (hunk 1 above)
Comment (junkil-park) [reply aptos-move/aptos-release-builder/data/release.yaml]: @kent-white confirmed that two discussion urls in one proposal would not work in UI. So, I added the links to the `description` field and kept only one link (for AIP-55) in the `discussion_url` field.
@davidiw , @movekevin , @perryjrandall , @runtian-zhou , please let me know if you have any comments or concern for this.

**Review by perryjrandall — APPROVED** [review]

**Review by movekevin — APPROVED** [review]

**Pull request merged**